src/applications/personalization/profile-2/msw-mocks.js

```diff
@@ -60,3 +60,353 @@ export const updateDirectDepositFailure = [
     );
   }),
 ];
+
+// When the transaction cannot be created at all
+export const addEmailAddressCreateTransactionFailure = [
+  rest.post(`${prefix}/v0/profile/email_addresses`, (req, res, ctx) => {
+    return res(
+      ctx.status(400),
+      ctx.json({
+        // This is an error response I saw when trying to start a phone number
+        // update, not email address update. I have not seen an error when
+        // trying to create an email address update transaction so am using this
+        // for the mock response
+        errors: [
+          {
+            title: 'Check Domestic Phone Number',
```

Comment:
Could we use an email error response from `config/locales/exceptions.en.yml` in `vets-api`?

Reply:
whoa... I did not know this file existed